Bitbucket.Authentication/Src/Authority.cs

```diff
@@ -69,7 +69,7 @@ public Authority(RuntimeContext context, string restRootUrl = null)
                 throw new ArgumentNullException(nameof(targetUri));
             if (credentials is null)
                 throw new ArgumentNullException(nameof(credentials));
-            if (resultType != AuthenticationResultType.Failure && resultType != AuthenticationResultType.Success && resultType != AuthenticationResultType.TwoFactor)
```

Comment:
Ditto, the comment above this doesn't appear to be diffing against microsoft:master…